### DISCUSSION TITLE
Can I use OpenSSL v 3.0.12?

### DISCUSSION THREAD

**Discussion by KMSiniline** [cesanta/mongoose#2801] [Ask questions, get answers] [answered]
I'm currently using Mongoose 7.14, can I use openssl 3.0.12 to work with WSS without any compatibility issues?

**Comment by scaprile** [comment, marked as answer]
Mongoose is tested daily with OpenSSL in whatever Github provides for their Ubuntu and MacOS versions.
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
https://github.com/cesanta/mongoose/discussions/2209
https://github.com/cesanta/mongoose/issues/2679
